src/bin/deku_node.ml

```diff
@@ -25,8 +25,12 @@ let handle_request (type req res)
       | Ok response ->
         let%await response = Parallel.encode E.response_to_yojson response in
         Dream.json response
-      | Error err -> raise (Failure (Flows.string_of_error err)))
-    | Error err -> raise (Failure err) in
+      | Error err ->
+        Dream.respond ~status:`Internal_Server_Error (Flows.string_of_error err)
```

Comment:
Do we expect that we log these errors somewhere else? Otherwise it would be good if we could log them here as well.